src/shared/reboot-util.c

```diff
@@ -36,6 +39,22 @@ int update_reboot_parameter_and_warn(const char *parameter) {
         return 0;
 }
 
+int get_reboot_parameter(char **parameter) {
```

Comment:
maybe call this `read_reboot_parameter()`, emphasizing that this does more than just a trivial get operation, but actually goes to the fs

Reply:
renamed as suggested